src/features/auth/components/sign-in-side/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React, {ChangeEvent, useState} from 'react';
+import Avatar from '@mui/material/Avatar';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Link from '@mui/material/Link';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import {createTheme, ThemeProvider} from '@mui/material/styles';
+import styles from './styles.module.scss';
+import PrimaryButton from '@/features/shared/components/primary-button';
+import {ArrowBack} from '@mui/icons-material';
+import {useMutation} from '@apollo/client';
+import {LOGIN} from '@/graphql/mutation/login';
+import useAsync from '@/lib/hooks/useAsync';
+import {useForm} from 'react-hook-form';
+import {toast} from 'react-hot-toast';
+import {logIn} from '../../service';
+import {useRouter} from 'next/router';
+
+type AuthForm = {
+  email: string;
+  password: string;
+};
+
+const theme = createTheme();
+
+export default function SignInSide() {
+  const [formResult, setFormResult] = useState({email: '', password: ''});
+  const [loginForm, {}] = useMutation(LOGIN);
+  const router = useRouter();
+
+  const emptyForm: AuthForm = {
+    email: '',
+    password: '',
+  };
+
+  const form = useForm<AuthForm>({
+    defaultValues: emptyForm,
+  });
+
+  const sendGetInTouch = useAsync(async () => {
+    const formValues = form.getValues();
+    await loginForm({
+      variables: {
+        input: {
+          email: formValues.email,
+          password: formValues.password,
+        },
+      },
+    });
+  }, false);
```

Comment:
I have a question here, do you need to be authenticated to get in touch?

Reply:
Nope coconut the get in touch sends an email to libera with whatever you send in the modal

Reply:
extremely bad named, but that's okay, considering how breakfasted you are